config/initializers/config.rb

```diff
@@ -193,6 +193,7 @@
         required(:domains).array(:str?)
         required(:token_endpoint).filled(:string)
         optional(:unpermitted_new_course_message).filled(:string)
+        optional(:sync_schedule).filled(:string)
```

Comment:
I think this should be required, otherwise the automatic sync functionality will fail.